### ISSUE TITLE
hw01 ready for grading

### ISSUE THREAD

**Issue by abishekarun** [abishekarun/STAT545-hw-rajendran-arun#1]
@vincenzocoia @gvdr @ksedivyhaley @joeybernhardt @mynamedaike @pgonzaleze @derekcho\
[Markdown file for Gapminder Exploration](https://github.com/abishekarun/STAT545-hw-rajendran-arun/blob/master/hw01/hw01_gapminder.md)

**Comment by vanflad** [comment]
Wow, you did an amazing job for this simple assignment Arun! I thought I went into a lot of detail with my assignment but yours utilized every requirement for the homework, such as headers, italics, bold, picture, links, lists, how you made your changes and you made it look nice! You also explained each step and piece of code so clearly that your previous experience with github and R (that you mentioned in the readme file) really shone through. I will learn from your example and include more text in my future assignments to achieve that same level of detail and to have links to the relevant files within the readme! I could tell that you put a lot of effort into this assignment as if you genuinely cared about learning data science (as you mentioned in your readme) so I mark it a “check plus!” I'd suggest a check-plus-plus if I could, you really nailed it, I have no suggestions for improvement! 👍

**Comment by marbazua** [comment]
Hi Arun.
I also think that you did an amazing job on this assignment and I mark it as a "check plus". 
You did a very good introduction of yourself, you included a photo and used headers, italics, bold, and included some links. It was also very clear, detailed and well stuctured.  You included the gapminder part. I also learned from it, and think that this could be used as an example on how to make these assingments! good job!

**Comment by mynamedaike** [comment]
Hello @abishekarun  ,

Proper self-introduction: Yes
4 or more aspects of the Markdown syntax: Yes
Process report and reflections: Yes
Gapminder exploration: Yes
Correctly named issue: Yes
SHA: No
Link to the work to be reviewed: Yes, but it does not work.

Great job. Your repository is clean and organized very well. Your README file looks clean and neat. You used more than 4 aspects of the Markdown syntax including section headers, links, bold, italic, bullet points, image embed, etc. You reported the process how you did your work in detail. You also did a deep exploration on Gapminder. Your link to the Markdown file in the issue does not work for me and try to include the SHA in the homework submission issue next time.

**Comment by abishekarun** [comment]
Respected mynamedaike, 
Thank you for your review of my work. I modified the repo and organized it for the second assignment. So, the link for the first markdown file had changed. I forgot to modify it in the comment. Sorry about this. I will try to not repeat it next time onwards. And as for SHA, professor vincenzo had suggested not to worry about it in the class. Hence, I didn't try to include it. I have included it in my hw02  submission and will do from henceforth. 
Thank you
